### DISCUSSION TITLE
暗黑破坏神无法进入游戏

### DISCUSSION THREAD

**Discussion by the0demiurge** [Gamuxorg/bbs#302] [游戏或网站运行反馈]
### 以下是推荐模板
游戏名称：
暗黑破坏神
操作系统：*（uname -a）*
Arch Linux x64
bug描述：*（具体说说发生了什么）*
创建人物后点继续就会弹窗报错
don't find buttons images file unassign skillsbtns.dc6
终端运行输出 ：*（如果有的话）*

自行尝试解决bug的方式和结果：*（如果有的话）*

**Comment by shouhuanxiaoji** [comment]
已上传一个新的打包，请尝试并来这里进行反馈，谢谢
diablo2_1.13c-g6_i386.appimage

**Comment by the0demiurge** [comment]
成功运行，谢谢：）
